tools_v2/rename.py

```diff
@@ -1,46 +1,79 @@
 import argparse
-import subprocess
-import struct
 import json
 import shutil
 import os
+from os.path import abspath, basename, dirname
 
 argparser = argparse.ArgumentParser(description='Rename the files.')
-argparser.add_argument('js_file', help='the js file to parse')
-argparser.add_argument('new_name', help='the new js file name (and path), without the .json')
+argparser.add_argument('js_file',
+                       help='the js file to parse, or a directory containing tl-data.json')
+argparser.add_argument('new_name',
+                       help='the new js file name (and/or path), without the .json')
 args = argparser.parse_args()
 
-jsfile = args.js_file
-if jsfile[0] != "/":
-    jsfile = os.getcwd() + "/" + jsfile
+# Example:
+#  python rename.py tl-data.json myname
+#   - renames whatever tl-data.json points to in cwd to myname, re-points tl-data.json at it.
+#  python rename.py oldname.json newname
+#   - renames oldname to newname
+#  python rename.py /some/path /new/path
+#   - renames whatever /some/path/tl-data.json points to, to /new/path/<name>.json (the name
+#     is taken from the contents of /some/path/tl-data.json)
```

Comment:
I got a failure when running from outside the tl-data.json directory:

python rename.py /tmp/ /tmp/test/
IOError: [Errno 2] No such file or directory: u'tl-data.13264.json'

I think this is because the python script didn't chdir into the /tmp/ directory?